IFC4x3/Properties/t/TunnelExcMethod_0IxY668fj68P9Wr33SXafb/Documentation.md

```diff
@@ -0,0 +1 @@
+Type of the tunnel construction method that was applied when the mapping was performed.
```

Comment:
What does it mean that "mapping was performed"?

Reply:
Changed to "Type of tunnel excavation method that was applied when the observation was made" in [ebc851b](https://github.com/bSI-InfraRoom/IFC-Specification/pull/589/commits/ebc851b49d47f6426cb687367b124ed31c9053cf). Also changed name to TunnelExcavationMethod. Still to do: Reuse PEnum from #570 when merged.